### DISCUSSION TITLE
Cannot successfully call /zones/forecast/ALC030/observations

### DISCUSSION THREAD

**Discussion by DemiMarie** [weather-gov/api#554] [Q&A] [answered]
`curl -H 'Accept: application/json' -H 'User-Agent: Demi test agent' https://api.weather.gov/zones/forecast/ALC030/observations` gives a 500 error.  How can I access this information?

**Comment by jah1007** [comment, marked as answer]
ALC030 is not a valid forecast zone. ALZ030 is valid, https://api.weather.gov/zones/forecast/ALZ030/observations.

**Reply by DemiMarie** [reply]
Thanks!  What is the best way to figure out which zones are valid?

**Reply by jah1007** [reply]
The best way to determine this would be to determine the local forecast office. You can do this by doing a points query like https://api.weather.gov/points/32.289,-86.4. Once you have the local office you can do an offices request like https://api.weather.gov/offices/BMX.

**Reply by JJKraw** [reply]
Also note that the /points query will return the forecast, county, and fire zones for the point in question.

**Reply by DemiMarie** [reply]
I see.  I was using a CSV file to obtain a zone from a county code and state name.
